util/cron/common-gasnet.bash

```diff
@@ -13,6 +13,7 @@ export GASNET_SPAWNFN=L
 export GASNET_ROUTE_OUTPUT=0
 export CHPL_GASNET_CFG_OPTIONS=--disable-ibv
 export CHPL_SYSTEM_PREDIFF=$CHPL_HOME/util/test/prediff-for-gasnet
+export CHPL_RT_MASTERIP=127.0.0.1
```

Comment:
A nit, but I think this line should be next to `export GASNET_SPAWNFN=L`, to make it more likely it will be removed if somebody copy/pasted this to run in real multilocale.

Reply:
Funnily enough, I originally had it there for that reason, but then thought "Well, it's a runtime rather than build-time setting, and it's the latest one, so maybe I should put it after these others…